libraries/chain/transaction.cpp

```diff
@@ -600,11 +599,11 @@ const vector<bytes>* packed_transaction::get_context_free_data()const {
 const bytes* maybe_get_context_free_data(const packed_transaction::prunable_data_type::none&, std::size_t) { return nullptr; }
 const bytes* maybe_get_context_free_data(const packed_transaction::prunable_data_type::partial& p, std::size_t i) {
    if( p.context_free_segments.size() <= i ) return nullptr;
-   return p.context_free_segments[i].visit(
+     return std::visit(
```

Comment:
Indentation looks wrong. It is diff throughout actually, but this one looks like it could cause confusion.

Reply:
Do we have clang format file that I could run on these changes?

Reply:
No, but we have various ones around. Maybe create a JIRA issue for someone to commit one to the repo.